screens/LoginScreen.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button
+} from 'react-native';
+import { Actions } from 'react-native-router-flux';
+
+const LoginScreen = () => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.welcome}>
+        Login Screen
+      </Text>
+      <Button title="Go to Signup" onPress={() => Actions.signup()} />
```

Comment:
Not sure if it's just me, when I navigate to signup page the process seemed to be a lot slower than other navigations.

Reply:
It is probably the animation. The animation between the Login and Signup screen is the same as the Alarm and Snooze screen. Are you seeing the same lag there also? It doesnt seem to be a lot slower on my device but this might become an issue if our users use older devices.

Reply:
I just pushed a change to remove the animation. See if that helps